packages/core/util/offscreenCanvasPonyfill.tsx

```diff
@@ -45,23 +43,25 @@ if (weHave.realOffscreenCanvas) {
   ImageBitmapType = window.ImageBitmap || self.ImageBitmap
 } else if (weHave.node) {
   // use node-canvas if we are running in node (i.e. automated tests)
-  const { createCanvas: nodeCreateCanvas, Image } = require('canvas')
-  createCanvas = nodeCreateCanvas
+  createCanvas = (...args) => {
+    // @ts-ignore
+    return nodeCreateCanvas(...args)
+  }
   createImageBitmap = async (canvas, ...otherargs) => {
     if (otherargs.length) {
       throw new Error(
         'only one-argument uses of createImageBitmap are supported by the node offscreencanvas ponyfill',
       )
     }
     const dataUri = canvas.toDataURL()
-    const img = new Image()
+    // @ts-ignore
+    const img = new nodeImage()
     return new Promise((resolve, reject) => {
       img.onload = () => resolve(img)
       img.onerror = reject
       img.src = dataUri
     })
   }
-  ImageBitmapType = Image
```

Comment:
ImageBitmapType now not available statically (due to trying not to depend on initialization order). There could be alternative ways e.g. we make the ponyfill return a callback that gives us the createCanvas callback perhaps, but this is a bit intrusive.